### ISSUE TITLE
Improve error message while trying to upload an existing package ID 

### ISSUE THREAD

**Issue by anangaur** [NuGet/NuGetGallery#7805]
From twitter:

If you push to NuGet.org a package with an ID that already exists, you'll get a very misleading message about your API key. The current error message seems to believe that you already own the package ID but might be using an incorrect API Key. The error message should indicate both reasons.

**Comment by loic-sharma** [comment]
The current message is:

> The specified API key is invalid, has expired, or does not have permission to access the specified package.

I don't think this message is terrible. Maybe we should add a documentation link and/or an NUXXX error code so that we can provide a page with helpful debugging help?

**Comment by anangaur** [comment]
I agree its not terrible but there is a report of a customer finding it in-sufficient. 

You may not be able to publish a package due to the following reasons:
1. You are not owner of the package
2. The API-key is invalid, or expired
3. The API-key is valid and you own the package but API-Key does not have perms (scope) for the specific package.

The current messaging seems to to suggest 2 and 3. May be making 1 more explicit may help too?  Or separate messages by inferring if the author owns a package ID or not.

**Comment by skofman1** [comment]
@anangaur , we you have a new text in mind, let us know an we will change the message. Super easy :)
